### PR TITLE
nov 18 draft

### PR DESCRIPTION
**Pull request by jluk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+    * Norway West
+* Bug Fixes
+  * Fixed an issue where certain cluster configurations did not auto-register Network, Compute, and Storage providers for the user.
+  * Fixed enforcement to require patch versions of node pools must be equal to or within 2 patch versions less than the control plane.
+  * Fixed error messages incorrectly stating a version was not supported to return proper errors detailing what validation was failed.
+  * Added retries to retrieve a managed resource group. Errors can be returned with `ResourceGroupNotFound` due to slow Azure Resource Manager (ARM) data replication when AKS tries to place new managed resources into the managed resource group.
+* Behavior Changes
+  * Added a label `control-plane=true` to the `kube-system` namespace
+* Component updates
+  * Metrics server has been updated to v3.0.5
```
Comment (robbiezhang) [inline CHANGELOG.md]: v0.3.5

Comment (jluk) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+    * Norway West
+* Bug Fixes
+  * Fixed an issue where certain cluster configurations did not auto-register Network, Compute, and Storage providers for the user.
+  * Fixed enforcement to require patch versions of node pools must be equal to or within 2 patch versions less than the control plane.
+  * Fixed error messages incorrectly stating a version was not supported to return proper errors detailing what validation was failed.
+  * Added retries to retrieve a managed resource group. Errors can be returned with `ResourceGroupNotFound` due to slow Azure Resource Manager (ARM) data replication when AKS tries to place new managed resources into the managed resource group.
+* Behavior Changes
+  * Added a label `control-plane=true` to the `kube-system` namespace
+* Component updates
+  * Metrics server has been updated to v3.5
```
Comment (robbiezhang) [inline CHANGELOG.md]: v0.3.5

Comment (jluk) [reply CHANGELOG.md]: second time is the charm

**Pull request merged**